explainer.md

```diff
@@ -343,24 +348,19 @@ ARIA attributes, they are never reflected. Similarly, Accessible Properties
 do not reflect equivalent implicit properties of their associated DOM elements,
 either.
 
-Specifically, all of the supported attributes on AccessibleNode return null
-initially. It doesn't matter if the associated DOM element has a computed
-role, or if you give it an explicit ARIA role or ARIA attributes - none of
-these are ever reflected in the AccessibleNode object.
-
 ```js
 button = document.createElement("button");
 button.innerHTML = "Next";
-console.log(button.accessibleNode.role);   // null
-console.log(button.accessibleNode.label);  // null
+console.log(button.accessibleNode.role);   // null, not "button"
```

Comment:
I still think it's confusing that we conflate ARIA reflection and computed properties here, and then discuss computed properties again below. Was there something specific you disagreed with about my suggested restructuring separating these two concerns altogether?